packages/celer-web-app/src/data/storage/RecentPages.test.ts

```diff
@@ -0,0 +1,57 @@
+import { addPageToRecents, loadRecentPages, MAX_RECENT_PAGES } from "data/storage";
```

Comment:
Should use relative imports if you are in the same package

 Don't need to export MAX_RECENT_PAGES after that